docs/source/configuration/logging.mdx

```diff
@@ -61,62 +70,440 @@ In both examples, the actual filter used by the router is as defined in the RUST
+3. `telemetry.logging.common.service_name` in `router.yaml`
+
+      <ExpansionPanel title="Example service_name">
+
+      Example setting service name in `telemetry.logging.common.service_name`:
+
+      ```yaml title="router.yaml"
+      telemetry:
+        logging:
+          common:
+            # (Optional) Set the service name to easily find logs related to the apollo-router in your metrics dashboards
+            service_name: "router" #highlight-line
+      ```
+
+      </ExpansionPanel>
+
+
+4. `telemetry.logging.common.resource` in `router.yaml`
+
+      <ExpansionPanel title="Example resource">
+
+      Example setting service name in `telemetry.logging.common.resource`:
 
-- [Basic logging](#basic-logging-via-shell), primarily for local development
-- [JSON-formatted logging](#json-formatted-logging), for compatibility with searchable logging tools like Google Cloud Logging
+      ```yaml title="router.yaml"
+      telemetry:
+        logging:
+          common:
+            resource:
+              # (Optional) Set the service name to easily find logs related to the apollo-router in your metrics dashboards
+              "service.name": "router" #highlight-line
+      ```
 
-The router uses basic logging whenever an interactive shell session is attached, and it uses JSON-formatted logging otherwise (e.g., in CI and deployed environments). You can also enforce a specific format in configuration.
+      </ExpansionPanel>
 
-### Basic logging via shell
+If the service name isn't explicitly set, then by default the service name will be set to `unknown_service:apollo_router` (or `unknown_service` if the executable name cannot be determined).
 
-Basic logging is mostly used during local development:
+### Resource attribute
 
+A resource attribute is a set of key-value pairs that provide additional information to an exporter. Application performance monitors (APM) may interpret and display resource information. 
+
+In `router.yaml`, resource attributes are set in `telemetry.logging.common.resource`. For example:
+
+```yaml title="router.yaml"
+telemetry:
+  logging:
+    common:
+      resource:
+        "environment.name": "production"
+        "environment.namespace": "{env.MY_K8_NAMESPACE_ENV_VARIABLE}"
 ```
-2022-03-18T10:28:11.090054Z  INFO apollo_router::executable: apollo-router@0.1.0-alpha.9
-2022-03-18T10:28:11.160650Z  INFO apollo_router: Starting Apollo Router
-2022-03-18T10:28:11.339906Z  INFO apollo_router: Listening on http://127.0.0.1:4000 🚀
+
+For OpenTelemetry conventions for resources, see [Resource Semantic Conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md).
+
+## Logging to stdout
+
+Logging in the Apollo Router can be directed to stdout.
+
+### Configure stdout
+
+#### `enabled`
+
+To enable stdout logging, set the `enabled` option to `true`:
+
+```yaml title="router.yaml"
+telemetry:
+  logging:
+    stdout:
+      enabled: true #highlight-line
```

Comment:
Where else can it be sent but to stdout? I mean, if we set `enabled`: `false` where does logging go? Does that just disable logging?

Reply:
It just disables logging.
Adding the enabled flag allows consistency with other exporters (all of which have enabled flags).

Eventually we may add support for the tracing log rotation functionality.